components/admin_console/group_settings/groups_list/groups_list.tsx

```diff
@@ -2,44 +2,74 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import {FormattedMessage} from 'react-intl';
+import {GroupSearchOpts} from 'mattermost-redux/types/groups';
 
-import * as Utils from 'utils/utils.jsx';
+import * as Utils from 'utils/utils';
 
-import GroupRow from 'components/admin_console/group_settings/group_row.jsx';
+import GroupRow from 'components/admin_console/group_settings/group_row';
 import NextIcon from 'components/widgets/icons/fa_next_icon';
 import PreviousIcon from 'components/widgets/icons/fa_previous_icon';
 import SearchIcon from 'components/widgets/icons/search_icon';
-import CheckboxCheckedIcon from 'components/widgets/icons/checkbox_checked_icon.jsx';
-
+import CheckboxCheckedIcon from 'components/widgets/icons/checkbox_checked_icon';
 import {Constants} from 'utils/constants';
 
 const LDAP_GROUPS_PAGE_SIZE = 200;
 
-const FILTER_STATE_SEARCH_KEY_MAPPING = {
+type PropActions = {
+    getLdapGroups: (page?: number, perPage?: number, opts?: GroupSearchOpts) => Promise<{}>;
```

Comment:
Non-blocking, but we usually don't separate the actions types out from the `Props` type.